### ISSUE TITLE
Bold claims in the introduction need citations

### ISSUE THREAD

**Issue by ayshih** [sunpy/sunpy-1.0-paper#97]
```
At the time of writing, the NASA Heliophysics System Observatory consists of
18 missions with nearly 200 different instruments.
```

Okay, I can count 18 missions (https://smd-prod.s3.amazonaws.com/science-red/s3fs-public/atoms/files/hso_fleet_chart_april_2019.pdf), but I'm skeptical about "nearly 200" instruments.  Which spacecrafts have 10+ instruments on them?  Is there a source that backs up that number?

```
These core packages [NumPy, SciPy, Matplotlib, and pandas] form the backbone
for hundreds of thousands of additional scientific Python packages.
```

Is there a source that backs up "hundreds of thousands"?  Even if it's technically true, would any reader care about more than an extremely small number (say, ten) of these packages?

**Comment by mbobra** [comment]
I made these bold claims. 

The NASA HSO missions in prime and extended operations include 129 instruments. They are enumerated below. Some of these are turned off but functional (e.g. SoHO/MDI), some are partly functional (e.g. SDO/EVE takes data, but SAM+MEGS-A are longer working), and some are lost (e.g. STEREO-B). You're right -- nearly 200 is an exaggeration. I had included Cluster in my previous count, which added 44 instruments.

Mission | Instruments 
--- | ---
Advanced Composition Explorer (ACE) | 9
Aeronomy of Ice in the Mesosphere (AIM) | 3
Geotail | 5
Gold | 1
Hinode | 3
Interstellar Boundary Explorer (IBEX) (pair of spacecraft) | 1 x 2
Interface Region Imaging Spectrograph (IRIS) | 1
Magnetospheric Multiscale Mission (MMS) (4 spacecraft) | 10 x 4
Solar Dynamics Observatory (SDO) | 3
Solar and Heliospheric Observatory (SoHO) | 12
Solar and Terrestrial Relations Observatory (STEREO) (pair of spacecraft) | 4 x 2
Parker Solar Probe | 5
THEMIS + ARTEMIS | 7 x 2
Thermosphere, Ionosphere, Mesosphere Energetics and Dynamics (TIMED) | 4
The Van Allen Probes (pair of spacecraft) | 5 x 2
Voyager 1 and 2 | 9 combined

Regarding the phrase "hundreds of thousands of packages": [PyPI](https://pypi.org/) currently lists 186,374 projects with 1,376,343 releases. I certainly use more than 10 in my analysis -- mpld3, pygal, bokeh, seaborn, scikit-learn, lime, tensorflow, torch, keras, dask, scikit-image, opencv, and simplecv to name a few. [PyHC](http://heliopython.org/projects/) lists 54 projects. So the relevant number is bigger than 10.
